moped-database/migrations/1704744986000_substantial_completion_date/down.sql

```diff
@@ -0,0 +1,624 @@
+--revert to version 1700515730257_fix_delete_component_bug
```

Comment:
i like this detail and it would be nice to have it for the `component_arcgis_online_view` view in this down migration too imo

Reply:
good call—thanks. it is done (and in this case is the same version as the `project_list_view`)